src/libcollections/vec.rs

```diff
@@ -1627,6 +1630,7 @@ impl<T> IntoIter<T> {
     #[inline]
     /// Drops all items that have not yet been moved and returns the empty vector.
     #[unstable(feature = "iter_to_vec")]
+    #[deprecated(since = "1.3.0", reason = "replaced by drain()")]
```

Comment:
Not for every use case, for example not for GroupByLazy, which would like to recycle allocations of the IntoIter it uses.